test/Interop/SwiftToCxx/stdlib/core-foundation-types-in-cxx.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -typecheck %s -typecheck -module-name UseCoreFoundation -enable-experimental-cxx-interop -clang-header-expose-decls=all-public -emit-clang-header-path %t/UseCoreFoundation.h
+// RUN: %FileCheck %s < %t/UseCoreFoundation.h
+
+// REQUIRES: OS=macosx
```

Comment:
Could you please do `// REQUIRES: objc_interop` to make this run on iOS/watchOS/etc?